plugins/woocommerce/tests/e2e-pw/tests/api-tests/orders/order-complex.test.js

```diff
@@ -95,17 +96,56 @@ test.describe( 'Orders API test', () => {
 				data: {
 					delete: ids,
 				},
+				failOnStatusCode: true,
 			} );
 		}
 
+		/**
+		 * Create the default "Reduced rate" and "Zero rate" tax classes if they're missing.
+		 * If successfully created, delete them at teardown to revert the list of tax classes.
+		 *
+		 * Allow 400 Bad Request as long as it's due to a duplicate tax class creation attempt.
+		 * If not, fail immediately.
+		 */
+		for ( const name of [ 'Reduced rate', 'Zero rate' ] ) {
+			const response_createTaxClasses = await request.post(
```

Comment:
Nitpick on variable naming `response_createTaxClasses` instead of `responseCreateTaxClasses`

Reply:
Thanks Adi! I renamed the variable just now in the next commit 757bfe5cc2620cf44655faf4c12c3c3d71005992.